theseus/geometry/tests/point_types_mypy_check.py

```diff
@@ -0,0 +1,17 @@
+import theseus as th
+
+
+def mypy_operations_check():
```

Comment:
For some reason `mypy` doesn't catch any errors if this function definition is here. We don't need this, so let's remove it.